Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla44944.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Bugzilla, 44944, "iOS: Text goes outside the bounds of Entry if it can't fit inside", PlatformAffected.iOS)]
+	public class Bugzilla44944 : TestContentPage
+	{
+		protected override void Init()
+		{
+			Content = new Grid
+			{
+				Children =
```

Comment:
Since this is a test that we can't easily automate (Test Cloud doesn't really have a good facility right now for determining whether the text is outside the bounds of the UITextView), we need to add on-screen instructions for a human verifying this fix. See https://github.com/xamarin/Xamarin.Forms/blob/dbd4390c5bf22a3f43a0f2f9b5defe4a76c727f4/Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla44584.cs for an example.